src/Microsoft.DotNet.XHarness.iOS.Shared/TestImporter/Templates/Managed/Resources/src/watchOS/Extension/InterfaceController.cs

```diff
@@ -140,6 +140,9 @@ void RunTests ()
 					case XmlVersion.NUnitV3:
```

Comment:
Again, I think here is a NUnitV3 default?

Reply:
It is already the default in https://github.com/dotnet/xharness/pull/102/files#diff-dc254c28cb157a76f159304a4405239bR134 but if we get a diff from the options we respect it.